requirements.txt

```diff
@@ -1,2 +1,3 @@
 beautifulsoup4==4.11.1
 selenium==4.2.0
+tqdm==4.64.1
```

Comment:
i would rather not introduce an extra dependency (see philosophy in `README.md`)